README.md

```diff
@@ -132,6 +132,12 @@ and many more ...
 - **What if I failed to sign in Amazon account over SSH remote development?**
     - Please refer to [Forwarding a port / creating SSH tunnel](https://code.visualstudio.com/docs/remote/ssh#_forwarding-a-port-creating-ssh-tunnel) to check whether you have forwarded port 9090.
 
+- **Why do I get a prompt for Git credentials when an Alexa host skill is created or downloaded with Git version 2.26.x or 2.25.x?**
```

Comment:
- typo : `Alexa hosted skill`. 
- Better to remove the `Git` information from the question here. In the answer itself we can ask them to check their `git` version by running the command. If it is one of the specific versions, then request them to update their git version. Otherwise we can suggest them to create an issue on the toolkit.